src/Services/DatabaseTools/AbstractDbalDatabaseTool.php

```diff
@@ -36,7 +36,7 @@ protected function getPlatformName(): string
         // AbstractMySQLPlatform was introduced in DBAL 3.3, keep the MySQLPlatform checks for compatibility with older versions
         if ($platform instanceof AbstractMySQLPlatform || $platform instanceof MySqlPlatform) {
             return 'mysql';
-        } elseif ($platform instanceof SqlitePlatform) {
+        } elseif ($platform instanceof SQLitePlatform) {
```

Comment:
```suggestion
        } elseif ($platform instanceof SqlitePlatform || $platform instanceof SQLitePlatform) {
```

Is this needed to keep compatibility?